.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,16 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
+**Description**
 
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
+<!--
+Reminder: If you decide to merge with any failing checks, add an explanatory comment before doing so.
 -->
 
-
 ---
 **PR checklist**
 
-- [ ] This PR meets the Acceptance Criteria in the JIRA story
```

Comment:
The intention here is to encourage an extra check of the AC before closing a story, to address a habit of doing **most** of a story, but missing some pieces.  TBH I continue to feel that's an area where we need to improve.

But this is not relevant to all PRs, so I'm not sure that it belongs here.

How about something like this: "if this PR is intended to complete a JIRA story, I have checked that all AC are met for that story"